lib/api/entities.rb

```diff
@@ -97,6 +97,12 @@ class Namespaces < Grape::Entity
       expose :id, documentation: { type: Integer, desc: "Namespace ID" }
       expose :clean_name, as: :name, documentation: { type: String, desc: "Namespace name" }
       expose :created_at, :updated_at, documentation: { type: DateTime }
+      expose :description, documentation: {
```

Comment:
I'm not entirely sure about this. Maybe for this version of the API we can leave this as internal ?

Reply:
Maybe, I dunno. I added it because it was necessary to display the description on the `show` page. I think we can keep this way, maybe it's useful for someone. It's an optional field, so if not used it returns empty. No big deal. Up to you.

Reply:
Sounds reasonable. Let's leave it like this :+1: